src/modules/UI/scenes/Plugins/EdgeProvider.js

```diff
@@ -288,6 +296,60 @@ export class EdgeProvider extends Bridgeable {
     Actions.pop()
   }
 
+  async getTransactions () {
+    // Get Wallet Info
+
+    const guiWallet = UI_SELECTORS.getSelectedWallet(this._state)
+    const coreWallet = CORE_SELECTORS.getWallet(this._state, guiWallet.id)
+    const currencyCode = UI_SELECTORS.getSelectedCurrencyCode(this._state)
+
+    // Prompt user with yes/no modal for permission
+
+    const confirmTxShare = await Airship.show(bridge => (
+      <TwoButtonSimpleConfirmationModal
+        bridge={bridge}
+        title={s.strings.fragment_wallets_export_transactions}
+        subTitle={sprintf(s.strings.transaction_history_permission, coreWallet.name)}
+        cancelText={s.strings.no}
+        doneText={s.strings.yes}
+      />
+    ))
+
+    // Grab transactions from current wallet
+
```

Comment:
The comment that describes the following block is a great coding style, but there shouldn't be a blank line between it and the block itself.